core/iwasm/fast-jit/fe/jit_emit_control.c

```diff
@@ -595,12 +595,17 @@ handle_next_reachable_block(JitCompContext *cc, uint8 **p_frame_ip)
         }
         else if (block->incoming_insns_for_end_bb) {
             *p_frame_ip = block->wasm_code_end + 1;
-            return handle_op_end(cc, p_frame_ip, false);
+            return handle_op_end(cc, p_frame_ip, block == first_block);
+        }
+        else if (LABEL_TYPE_FUNCTION == block->label_type) {
+            /* reach the outest block */
+            *p_frame_ip = block->wasm_code_end + 1;
```

Comment:
It is OK, but a little strange, the blocks are destroyed when destroying cc.
Had better merge it into else branch:
```
else {
    if (LABEL_TYPE_FUNCTION == block->label_type)
            /* reach the outest block */
            *p_frame_ip = block->wasm_code_end + 1;
    ....
}
```